### DISCUSSION TITLE
Is there a way for putting a single package in pre mode?

### DISCUSSION THREAD

**Discussion by gfellerph** [changesets/changesets#1068] [Q&A] [answered]
I am maintaining a design system monorepo (https://github.com/swisspost/design-system) and we have a couple of packages already published. When we add a new package to the bunch, we'd like to release the new one as a pre-release before we're sure it's ready for prod. In the meantime, the other packages need patches or get new features and we're writing changesets for them and like to release them in non-pre mode.

Is it possible to use your awesome github action to release both pre-release packages and regular release packages on the same branch (main)?

(There are similar questions, but I found it hard to figure out if they cover the exact use case we have here)

**Comment by Andarist** [comment, marked as answer]
Unfortunately, it's not possible today. It could be implemented - but requires a significant design and implementation effort. It's something I'd like to support in the future but I have no idea when that might be.

You could script this though... if you would just juggle the directories and the workspaces config a little bit.
